source/builders.txt

```diff
@@ -0,0 +1,116 @@
+.. _kotlin-sync-builders:
+
+===============================
+Implement Builders Code Pattern
+===============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. .. toctree::
+
+..    /fundamentals/builders/aggregates
+..    /fundamentals/builders/filters
+..    /fundamentals/builders/indexes
+..    /fundamentals/builders/projections
+..    /fundamentals/builders/sort
+..    /fundamentals/builders/updates
+
+Overview
+--------
+
+This page describes how to implement the builders pattern in your code and describes
+benefits of using builder classes.
+
+The {+driver-short+} provides type-safe builders classes and methods that enable developers to
+efficiently build queries and aggregations. To learn more about the builders pattern,
+see `Type-safe builders <https://kotlinlang.org/docs/type-safe-builders.html>`__ in the
+{+language+} documentation.
```

Comment:
S: possible convention - where we use "builder classes" use singular builder. Where we refer to builders alone, use plural
```suggestion
The {+driver-short+} provides type-safe builder classes and methods that enable developers to
efficiently build queries and aggregations. To learn more about the builders pattern,
see `Type-safe builders <https://kotlinlang.org/docs/type-safe-builders.html>`__ in the
{+language+} documentation.
```